reviews/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from reviews.models        import Review
+from django.http           import JsonResponse, request
+from django.views          import View
+from users.models          import User
+from products.models       import Product
+from users.utils           import login_decorator
+
+class ReviewPostingView(View):
+    @login_decorator    
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            if data["title"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_REVIEWS"}, status = 400)
+
+            if data["size_rating"] == "" or data["color_rating"] == "" or data["delivery_rating"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_RATING"}, status = 400)
+
+            if data["size_rating"] >= 6 or data["color_rating"] >= 6 or data["delivery_rating"] >= 6 :
+                return JsonResponse({"MESSAGE":"RATING_INPUT_ERROR"}, status = 400)
+
+            user = User.objects.get(id = request.user.id)
+            Review.objects.create( 
+                user            = User.objects.get(id=request.user.id),
```

Comment:
```suggestion
            
            Review.objects.create( 
                user            = request.user,
```
이미 데코레이터를 통해서 유저 정보를 알고 있기 때문에 다시 한 번 더 유저를 안 가져와도 됩니다.